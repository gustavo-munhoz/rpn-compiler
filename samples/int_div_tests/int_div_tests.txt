(6 2 /)
(10 3 /)
(10 5 /)
(7 3 /)
(1 2 /)
(-10 3 /)
(10 -3 /)
(-10 -3 /)
(-7 3 /)
(0 5 /)
(0 -5 /)
(80 15 /)
(-80 15 /)
(65504 2 /)
(65504 1 /)
(-1 2 /)
(0.9 1 /)
(-0.9 1 /)