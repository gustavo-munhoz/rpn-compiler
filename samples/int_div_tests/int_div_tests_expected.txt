0x4200
0x4200
0x4000
0x4000
0x0000
0xC200
0xC200
0x4200
0xC000
0x0000
0x8000
0x4500
0xC500
0x77FF
0x7BFF
0x8000
0x0000
0x8000